docs/release-notes/highlights-2.15.0.asciidoc

```diff
@@ -0,0 +1,21 @@
+[[release-highlights-2.15.0]]
+== 2.15.0 release highlights
+
```

Comment:
Should we add information about the role mapping fix in 8.16 (as a known issue maybe), or is that not strictly related to this ECK release?

Reply:
I would probably still include the existing known issue. And we need to amend it to mention that the issue is resolved in 8.16. We can do that in a separate PR though.